sdk/search/Azure.Search.Documents/src/SearchClientOptions.cs

```diff
@@ -32,28 +32,23 @@ public enum ServiceVersion
             V2020_06_30 = 1,
 
             /// <summary>
-            /// The 2021_04_30_Preview version of the Azure Cognitive Search service.
+            /// The 2023_11_01 version of the Azure Cognitive Search service.
             /// </summary>
-            V2021_04_30_Preview = 2,
-
-            /// <summary>
-            /// The 2023_10_01_Preview version of the Azure Cognitive Search service.
-            /// </summary>
-            V2023_10_01_Preview = 3,
-#pragma warning restore CA1707
+            V2023_11_01 = 2,
+            #pragma warning restore CA1707
         }
 
         /// <summary>
         /// The Latest service version supported by this client library.
         /// </summary>
-        internal const ServiceVersion LatestVersion = ServiceVersion.V2023_10_01_Preview;
+        internal const ServiceVersion LatestVersion = ServiceVersion.V2023_11_01;
 
         /// <summary>
         /// The service version to use when creating continuation tokens that
         /// can be passed between different client libraries.  Changing this
         /// value requires updating <see cref="Azure.Search.Documents.Models.SearchContinuationToken"/>.
         /// </summary>
-        internal const ServiceVersion ContinuationTokenVersion = ServiceVersion.V2020_06_30;
+        internal const ServiceVersion ContinuationTokenVersion = ServiceVersion.V2023_11_01;
```

Comment:
Did you change this algorithm?  If you didn't touch the continuation token parsing code, then we should leave the 2020 value.

Reply:
No, I didn't. Reverted back to 2020 value.